sources/demo/template.html

```diff
@@ -237,9 +238,7 @@ <h4>Icon names</h4>
 <div class="icons">
 {{#each icons}}
   <div>
-    <svg width="24" height="24" class="gridicon {{name}}">
-    <use xlink:href="gridicons.svg#{{name}}" />
```

Comment:
`xlink:href` was deprecated and is not supported in modern html. Additionally, plain `href` does not support importing from files any more. It appears that `<object data="` is the way to go, but hard to verify since the file wasn't working on trunk.